app/jobs/checksum_report_job.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'faraday'
```

Comment:
I don't believe you need this here. It should already be required on line 14 of `config/application.rb`.

Reply:
done